job_definitions/notify_buyers_to_award_closed_briefs.yml

```diff
@@ -0,0 +1,81 @@
+{% set environments = ['production'] %}
+---
+{% for environment in environments %}
+{% with vars = [
```

Comment:
Maybe this could have a better name than `vars`/`v`. Also any reason you opted to use `with` as opposed to the `set` above?

Reply:
I couldn't think of a better name! `jobs` seems a bit vague, how about `reminders`? 

Re: `with`: no reason at all! I will change to `set` for consistency.